gui/UsernamesAndPasswords.java

```diff
@@ -0,0 +1,110 @@
+package com.dyn.admin.gui;
```

Comment:
It is blank because we aren't attached to the database. but the username will appear to the left of the colon and the password on the right.
![usernameandpassword](https://cloud.githubusercontent.com/assets/11052852/15722489/9e8840f8-2803-11e6-8eac-36715e545e06.PNG)